modules/mysql/tests/test_mysql.py

```diff
@@ -30,6 +30,17 @@ def test_docker_run_mysql_8():
                 assert row[0].startswith("8")
 
 
+# @pytest.mark.skipif(is_arm(), reason="mysql container not available for ARM")
+def test_docker_run_mysql_8():
+    config = MySqlContainer("mysql:8")
+    with config as mysql:
+        engine = sqlalchemy.create_engine(mysql.get_connection_url())
+        with engine.begin() as connection:
+            result = connection.execute(sqlalchemy.text("select version()"))
+            for row in result:
+                assert row[0].startswith("8")
+
+
```

Comment:
probably not intended, it touches `mysql` :slightly_smiling_face:

Reply:
Yupp. I don't have push to David's repo, but I took the liberty of opening a PR on a fork that David got from https://github.com/testcontainers/testcontainers-python/pull/363.

Reply:
it is also unintended because it is an exact copy of the preceding lines